template/error.go

```
@@ -45,3 +45,10 @@ var incorrectTemplatingError = errgo.New("incorrect templating")
 func IsIncorrectTemplating(err error) bool {
 	return errgo.Cause(err) == incorrectTemplatingError
 }
+
+var emptyKubernetesResourcesDirectoryPath = errgo.New("empty kubernetes resources directory path")
```

`emptyDirectoryPathError` would work for me. You could then mask the error with additional information ala `microerror.MaskAnyf(emptyDirectoryError, "kubernetes resource '%s'", thePath)`.